applications/clouddemo/netcore/CloudDemo.MvcCore/Controllers/HomeController.cs

```diff
@@ -1,5 +1,5 @@
 ﻿//
-// Copyright 2020 Google LLC
+// Copyright 2022 Google LLC
```

Comment:
We don't change copyright headers with changes. They remain the same for the lifetime of the file.